File: harmonic.java
public class harmonic {
    public static void main(String[] args) {
	double a = 1;
	double b = 0;
	
	while(true) {
	    for (int i = 0; i < 10000000; i++) {
		b += 1.0 / a;
		a ++;
	    }
	    System.out.println(b);
	}
    }
}

File: harmonic.f
C HARMONIC SERIES - WHAT IS THE SUM?
      DOUBLE PRECISION A,B
      A = 1.0
      B = 0.0
      DO 1
         DO 2 I=0,10000000,1
            B = B + 1 / A
            A = A + 1
 2       CONTINUE
         WRITE(6, 3) B
 1    CONTINUE
 3    FORMAT(D10.5)
      END
